packages/sdk-auth/src/tokenProvider.js

```diff
@@ -90,14 +113,22 @@ export default class TokenProvider {
       .then(
         (tokenInfo: TokenInfo): void => {
           newTokenInfo = tokenInfo
-          newTokenInfo.refresh_token = oldTokenInfo?.['refresh_token']
+          if (oldTokenInfo?.['refresh_token'])
```

Comment:
why not `oldTokenInfo?.refresh_token` like in the assignment below? Looks a bit strange with two different ways of calling the same.

Reply:
Flow was complaining that the property should not be in underscore notation. I am not sure why it complained only in this case.

Reply:
ahhh, ok, then I guess we can just leave it there.